client/exec.go

```diff
@@ -152,14 +152,10 @@ func (client *Client) Exec(opts *ExecOptions) (*ExecProcess, error) {
 	headers := map[string]string{
 		"Content-Type": "application/json",
 	}
-	resultBytes, changeID, err := client.doAsyncFull("POST", "/v1/exec", nil, headers, &body)
-	if err != nil {
-		return nil, err
-	}
 	var result execResult
-	err = json.Unmarshal(resultBytes, &result)
+	reqResponse, err := client.doAsync("POST", "/v1/exec", nil, headers, &body, &result)
```

Comment:
s/reqResponse/resp/

This PR is using it both ways. Let's please establish `resp` as the pattern as it's a nice complement to `req`.

Reply:
Done.